CHANGELOG.md

```diff
@@ -4,36 +4,23 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-#### [v1.1.58](https://github.com/PrefectHQ/orion-design/compare/v1.1.57...v1.1.58)
+#### [v1.1.59](https://github.com/PrefectHQ/orion-design/compare/v1.1.51...v1.1.59)
 
+- bump graphs version and tweek timeline [`#1033`](https://github.com/PrefectHQ/orion-design/pull/1033)
+- Chore: Make sure delete is the last menu item in the FlowMenu [`#1032`](https://github.com/PrefectHQ/orion-design/pull/1032)
 - Chore: Fix the type for provided components [`#1031`](https://github.com/PrefectHQ/orion-design/pull/1031)
-
-#### [v1.1.57](https://github.com/PrefectHQ/orion-design/compare/v1.1.55...v1.1.57)
-
-> 10 January 2023
-
```

Comment:
@brandonreid / @znicholasbrown  what's with this crazy changelog diff?

Reply:
Looks like we dropped the `v` in our version headings here, not sure why

Reply:
It also seems to drop some releases entirely